components/script/dom/webidls/XRInputSource.webidl

```diff
@@ -19,7 +19,7 @@ enum XRTargetRayMode {
 [SecureContext, Exposed=Window, Pref="dom.webxr.enabled"]
 interface XRInputSource {
   readonly attribute XRHandedness handedness;
-  // [SameObject] readonly attribute XRTargetRayMode targetRayMode;
+  readonly attribute XRTargetRayMode targetRayMode;
```

Comment:
Why no `[SameObject]`?

Reply:
Can't SameObject enums. I believe this was fixed upstream in the spec, we just never updated the commented webidl.